src/components/BulkEnrollmentPage/stepper/DismissibleCourseWarning.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 
-import { Alert, WarningFilled } from '@edx/paragon';
+import { Alert } from '@edx/paragon';
+import { WarningFilled } from '@edx/paragon/icons';
```

Comment:
There was an error in console about `WarningFilled` not existing as an export in `@edx/paragon`, which makes sense because the icons are exported only at `@edx/paragon/icons`.